From: Matthias Fehring <buschmann23@opensuse.org>
Date: 2019-08-16 08:16:00 +0200
Subject: fix reading description files
Upstream: merged (https://bitbucket.org/flyingcircus/pycountry/pull-requests/26/read-readme-history-using-utf-8-encoding/diff)

When reading the description in setup.py the wrong encoding will
be selected for the text files.

---
 setup.py |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- a/setup.py	2019-07-15 16:35:49.000000000 +0200
+++ b/setup.py	2019-08-16 08:12:49.493720089 +0200
@@ -3,6 +3,8 @@
 # Copyright 2015- (c) Flying Circus Internet Operations GmbH
 # See also LICENSE.txt

+from io import open
+
 from setuptools import setup, find_packages


@@ -14,8 +16,8 @@
     description='ISO country, subdivision, language, currency and script '
                 'definitions and their translations',
     long_description=(
-        open('README.rst').read() + '\n' +
-        open('HISTORY.txt').read()),
+        open('README.rst', encoding='utf-8').read() + '\n' +
+        open('HISTORY.txt', encoding='utf-8').read()),
     license='LGPL 2.1',
     keywords='country subdivision language currency iso 3166 639 4217 '
              '15924 3166-2',
